library/Icingadb/ProvidedHook/Reporting/SlaReport.php

```diff
@@ -188,6 +188,12 @@ public function initConfigForm(Form $form)
             'min'         => '1',
             'max'         => '12'
         ]);
+
+        $form->addElement('checkbox', 'csv-json-export', [
```

Comment:
For field names we always use `snake_case`. I would also suggest to rename the field to `export_total`.